Switching from old "main" branch to new "abels" branch which (#21)
ivadomed/model-canal-seg: README.md

# Automatic segmentation of spinal canal

<p align="center">
  <img src="https://github.com/ivadomed/model-canal-seg/blob/abels/assets/canal_seg_visual.gif" alt="Segmentation on whole-spine image viewed on axial and sagittal planes" width="500">
</p>

This repository contains the code for deep learning-based segmentation of the spinal canal. 
The code is based on the [nnUNet framework](https://github.com/MIC-DKFZ/nnUNet).

> The spinal canal as here been defined using the anatomical boundary of the dural sac. The model has been trained to segment both spinal cord and CSF inside the dural sac. 

Reviewer: ```suggestion
The spinal canal was defined using the anatomical boundary of the dural sac. The model was trained to segment all the structures within the dural sac, including the spinal cord, cerebrospinal fluid (CSF), and nerve rootlets.
```

## Model Overview

> The model is a 3d nnUNet which was trained specifically on T2-weighted images and provides and provides a segmentation of the spinal canal. 

Reviewer: ```suggestion
The model is a 3D nnUNet, which was trained on T2-weighted images to segment the spinal canal. 
```

## How to use the model

### Install dependencies

> - [Spinal Cord Toolbox (SCT) v6.5](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/6.2) or higher -- follow the installation instructions [here](https://github.com/spinalcordtoolbox/spinalcordtoolbox?tab=readme-ov-file#installation)

Reviewer: ```suggestion
- [Spinal Cord Toolbox (SCT) v6.5](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/6.5) or higher -- follow the installation instructions [here](https://github.com/spinalcordtoolbox/spinalcordtoolbox?tab=readme-ov-file#installation)
```

Author: Sure! I should definitively have not forgot that !
